components/page.jsx

```diff
@@ -77,7 +77,11 @@ var Page = React.createClass({
           <Footer/>
         </div>
 
-        <ReactCSSTransitionGroup transitionName="modal">
+        <TimeoutTransitionGroup transitionName="modal"
+                                enterTimeout={0}
+                                leaveTimeout={0}
```

Comment:
Hmm, I am surprised this is `0`... I would have thought that it should be set to about `250`, since our CSS transitions in `modal.less` are `0.25s`?  Or am I wrong?